decidim-api/lib/decidim/api.rb

```diff
@@ -42,5 +42,11 @@ def self.add_orphan_type(type)
       @orphan_types ||= []
       @orphan_types += [type]
     end
+
+    # Public Setting that can make the API authentication necessary in order to
```

Comment:
I would move this config_accessor at line ~ 30  to have all the settings grouped.